### DISCUSSION TITLE
What can be the actual usecase of using this library to create pdf rather than doing normally

### DISCUSSION THREAD

**Discussion by Greatz08** [QuestPDF/QuestPDF#876] [General]
Please give some real life examples and explain in easy way so that I can understand how much beneficial it can be in many possible cases.

**Comment by girlpunk** [comment]
What do you mean by "doing normally"?

**Reply by Greatz08** [reply]
@girlpunk  by normal i mean writing in docs and converting to pdf or more advanced plus easy way using figma to create design and then convert to docs and edit and convert to pdf or directly convert to pdf and edit  :-)) these options look much more simpler in my opinion and time saving too

**Reply by girlpunk** [reply]
QuestPDF is a library for automatic PDF generation using it's layout engine. It's designed for integration into existing .Net programs which would not be possible with docs, Figma, etc.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
